test/constants.js

```diff
@@ -4,7 +4,8 @@ const constants = {
 		RAF_THRESHOLD : 16,
 		SMALL: 5
 	},
-	ITEM_TO_OBSERVE: 5
+	ITEM_TO_OBSERVE: 5,
+	SKIP_CYCLE: 3
```

Comment:
Yes. Also, `SKIP_CYCLE` is 3 and `NUM_SKIP_CYCLES` is 4. Shouldn't they be the same?